### ISSUE TITLE
Not possible to customise status bar via keyboard

### ISSUE THREAD

**Issue by isidorn** [microsoft/vscode#92149]
Currently you can customise the content of the status bar via mouse context menu.
However it is not possible to do this using just the keyboard. Or I am missing something?
We should just have a command that invokes the context menu on the status bar so the users can choose what is shown there.

**Comment by bpasero** [comment]
Partly related to the fact that you cannot focus status bar items, so you cannot even invoke their default command.

Up for grab if someone wants to chime in.

**Comment by ajborka** [comment]
In Insider's 1.43 and later, NVDA and JAWS can access status bar customization features by pressing the applications or shift+f10 keys on the keyboard when the screen reader's navigation cursor is positioned on the status bar. This also works with Orca screen reader in Ubuntu 20.04. I assume it would work in Ubuntu 18.04 and later, but haven't tested that fact. Thus, this issue can be closed if required.

**Comment by Jawz84** [comment]
@bpasero this issue has become part of the following coding challenge: https://www.microsoft.com/en-gb/business/azure/oss/?ocid=AID3010439_TWITTER_oo_spl100001210204923
It might need to be reopened.

**Comment by Jawz84** [comment]
@bpasero this issue has become part of the following coding challenge: https://www.microsoft.com/en-gb/business/azure/oss/?ocid=AID3010439_TWITTER_oo_spl100001210204923
It might need to be reopened.

**Comment by isidorn** [comment]
This is fixed now. The authors of that challenge should only look at open help wanted issues. Thanks
